dtls.c

```diff
@@ -1163,9 +1168,6 @@ dtls_update_parameters(dtls_context_t *ctx,
     data += sizeof(uint16);
   }
 
-  /* skip remaining ciphers */
-  data += i;
```

Comment:
I think that this is necessary here as `data` needs to point to the start of the compression methods below.

Reply:
Yes, I overseen the "ok" case.